electricitymap/contrib/lib/models/event_lists.py

```diff
@@ -279,6 +279,35 @@ def update_production_breakdowns(
 
         return production_breakdowns
 
+    @staticmethod
+    def filter_only_zero_production(
```

Comment:
Do we really want this as an extra function?
I would expect this to be the default behaviour of the ProductionBreakdownList with the possibility to override it for zones that can have 0 production like AX and US-FLA-HST.

Reply:
This is meant as a temporary fix that would then be handled by the outlier detection pipeline

Reply:
Would it really need to be handled by the outlier detection though?
We already filter out events that are all None so I would just extend on that with a boolean flag to turn it off.

Reply:
The whole point of the outlier detection is to ingest all data points and then flag them rather than filtering them before the database

Reply:
Does that mean we want to start returning all the null values as well?

I was under the assumption that we were going to filter out all clear errors as close to the parsers as possible and use the outlier detection to handle suspicious events that might not be clear errors.

Reply:
@VIKTORVAV99, the long term vision is that we do not do any kind of filtering at the parser level. We rather want to ingest everything we can because we want to be able to report on why we dropped certain parser events, also analyze the frequency at which they appear and other things which we are not able to do if it's filtered at parser level.

Right now we do not have a sufficiently mature infrastructure to handle all edge cases so we do this filtering at this level but all of this should be handled in the future outside of the parser hence why a central function which we can remove later on is probably the best place. Also better to not introduce too much complexity.

Reply:
> why a central function which we can remove later on is probably the best place.
This is what I mean, we already do this filtering in one place with the validator decorators. Would it not be better to extend that in some way so we can remove them in one place later instead of every parser we decide to use this function?